pkg/lib/lock.go

```diff
@@ -76,6 +76,7 @@ func Luksify(label string, logger types.KairosLogger, argsCreate ...string) (str
 	device := fmt.Sprintf("/dev/%s", part)
 
 	extraArgs := []string{"--uuid", uuid.NewV5(uuid.NamespaceURL, label).String()}
+	extraArgs = append(extraArgs, "--label", label)
```

Comment:
You can label the encrypted partition too! How didn't we think of it before? :D Good thinking @Itxaka

Reply:
yeah....

plus it would have shown the other issues before this....weird.

Plus thnkas to udev the /dev/disk/by-label gets updated once unlocked to point COS_PERSISTENT or whatever label to the unlocked so the rest works as normal....